homeassistant/components/services.yaml

```diff
@@ -1,5 +1,13 @@
 # Describes the format for available component services
 
+conversation:
```

Comment:
We shouldn't add more services to `services.yaml`. Instead, move `conversation.py` to `conversation/__init__.py` and create a new `conversation/services.yaml`